api/db/export/all.go

```diff
@@ -11,7 +11,7 @@ type all struct {
 	Plugins []plugins.Plugin `json:"plugins"`
 }
 
-func All(usersWPassword bool, outputFile string) error {
+func All(pretty bool, usersWPassword bool, outputFile string) error {
```

Comment:
Why put it as first parameter?
I think it would've made more sense as last parameter, just after `outputFile`, since it is the output file which will be pretty.

Reply:
No problem, I was not sure about this ! 👍